packages/govuk-frontend/src/govuk/components/summary-list/template.njk

```diff
@@ -61,12 +64,12 @@
           {% if row.actions.items.length %}
             <dd class="govuk-summary-list__actions {%- if row.actions.classes %} {{ row.actions.classes }}{% endif %}">
               {% if row.actions.items.length == 1 %}
-                {{ _actionLink(row.actions.items[0]) | indent(16 if params.card else 12) | trim }}
+                {{ _actionLink(row.actions.items[0], params.card.title) | indent(16 if params.card else 12) | trim }}
               {% else %}
                 <ul class="govuk-summary-list__actions-list">
                   {%- for action in row.actions.items -%}
                     <li class="govuk-summary-list__actions-list-item">
-                      {{- _actionLink(action) | indent(22 if params.card else 18) | trim -}}
+                      {{- _actionLink(action, params.card.title) | indent(22 if params.card else 18) | trim -}}
```

Comment:
Mind if we re-assess these indents and empty space removal?

Rendered HTML indentation is a bit strange

```html
<dd class="govuk-summary-list__actions">
    <ul class="govuk-summary-list__actions-list"><li class="govuk-summary-list__actions-list-item"><a class="govuk-link" href="#">Edit<span class="govuk-visually-hidden"> name</span>
            <span class="govuk-visually-hidden"> (Undergraduate teaching assistant)</span>
            </a></li><li class="govuk-summary-list__actions-list-item"><a class="govuk-link" href="#">Delete<span class="govuk-visually-hidden"> name</span>
            <span class="govuk-visually-hidden"> (Undergraduate teaching assistant)</span>
            </a></li></ul>
</dd>
```

Reply:
I've ended up crunching the entire `<ul>` and its contents onto a single line at the moment. It's not great, but as the links contain semantically significant whitespace, it seemed like the simplest 'tidy' solution.

The whole component is a jumble of weird indentation (it switches from 2 spaces to 4 at various points) so it's not something I want to focus too much on trying to fix in this PR, but certainly a nice thing to try and achieve here and in other components at some point.

Reply:
Thanks @querkmachine

For reference the example above is now:

```html
<dd class="govuk-summary-list__actions">
    <ul class="govuk-summary-list__actions-list"><li class="govuk-summary-list__actions-list-item"><a class="govuk-link" href="#">Edit<span class="govuk-visually-hidden"> name (Undergraduate teaching assistant)</span></a></li><li class="govuk-summary-list__actions-list-item"><a class="govuk-link" href="#">Delete<span class="govuk-visually-hidden"> name (Undergraduate teaching assistant)</span></a></li></ul>
</dd>
```

Not related to this comment but worth a mention, the title wrapper similarly is:

```html
    <div class="govuk-summary-card__title-wrapper"><h2 class="govuk-summary-card__title">Undergraduate teaching assistant</h2><ul class="govuk-summary-card__actions"><li class="govuk-summary-card__action">  <a class="govuk-link" href="#">Delete job history<span class="govuk-visually-hidden"> (Undergraduate teaching assistant)</span></a>
</li><li class="govuk-summary-card__action">  <a class="govuk-link" href="#">Withdraw job history<span class="govuk-visually-hidden"> (Undergraduate teaching assistant)</span></a>
</li></ul>
</div>
```